geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxy.java

```diff
@@ -980,15 +979,6 @@ private void closeOtherTransientFields() {
     // Commented to fix bug 40259
     // this.clientVersion = null;
     closeNonDurableCqs();
-
```

Comment:
These changes are reverting Barry's fix to log out the subject here, we are logging out the subject in the `close` method later on. So these changes are not needed here anymore.